pfb-analysis/scripts/export_connectivity.sh

```diff
@@ -58,36 +59,56 @@ function ec_export_table_shp() {
 
 function ec_export_table_csv() {
     OUTPUT_DIR="$1"
-    EXPORT_TABLENAME="$2"
+    OUTPUT_FILE_PREFIX="${2}"
+    EXPORT_TABLENAME="$3"
 
-    FILENAME="${OUTPUT_DIR}/${NB_OUTPUT_FILE_PREFIX}${EXPORT_TABLENAME}.csv"
+    FILENAME="${OUTPUT_DIR}/${OUTPUT_FILE_PREFIX}_${EXPORT_TABLENAME}.csv"
     psql -h "${NB_POSTGRESQL_HOST}" \
          -U "${NB_POSTGRESQL_USER}" \
          -p "${NB_POSTGRESQL_PORT}" \
          -d "${NB_POSTGRESQL_DB}" \
-         -c "COPY ${EXPORT_TABLENAME} TO '${FILENAME}' WITH (FORMAT CSV, HEADER)"
+         -c "\COPY ${EXPORT_TABLENAME} TO '${FILENAME}' WITH (FORMAT CSV, HEADER)"
 }
 
 if [ "${BASH_SOURCE[0]}" = "${0}" ]
 then
-    if [ "${1:-}" = "--help" ] || [ -z "${1:-}" ]
+    if [ "${1}" = "--help" ]
+    then
+        ec_usage
+    elif [ -z "${1}" ] || [ -z "${2}" ]
     then
         ec_usage
+        exit 1
     else
         echo "Exporting analysis to ${NB_OUTPUT_DIR}"
         NB_OUTPUT_DIR="${1}"
+        NB_OUTPUT_FILE_PREFIX="${2}"
         mkdir -p "${NB_OUTPUT_DIR}"
 
         # Export neighborhood_ways as SHP
-        ec_export_table_shp "${NB_OUTPUT_DIR}" "neighborhood_ways"
+        ec_export_table_shp "${NB_OUTPUT_DIR}" "${NB_OUTPUT_FILE_PREFIX}" "neighborhood_ways"
 
         # Export neighborhood ways as GeoJSON
         # TODO: Add ogr2ogr and try again
 
         # Export neighborhood_connected_census_blocks as SHP
-        ec_export_table_shp "${NB_OUTPUT_DIR}" "neighborhood_connected_census_blocks"
+        ec_export_table_shp "${NB_OUTPUT_DIR}" "${NB_OUTPUT_FILE_PREFIX}" "neighborhood_connected_census_blocks"
```

Comment:
Let's disable the connected_census_blocks export for now, its a massive file and doesn't hold much value.